tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala

```diff
@@ -43,7 +44,9 @@ object FgBioTask {
   *
   * @param compressionLevel the compress level to use for HTSJDK.
   */
-abstract class FgBioTask(var compressionLevel: Option[Int] = None)
+abstract class FgBioTask(var compressionLevel: Option[Int] = None,
+                         val asyncIo: Option[Boolean] = None,
+                         val tmpDir: Option[DirPath] = None)
```

Comment:
Want to add `logLevel` while you're at it?

Reply:
Want to submit a PR to my PR?  Joking.